docs/python/generators/examples_page.py

```diff
@@ -43,6 +44,44 @@ def collect_srcs(top):
                             yield Path(root) / f
 
             srcs = list(collect_srcs(root))
+            # deduplicate source file list while preserving order
+            seen_srcs = set()
+            deduped_srcs = []
+            for p in srcs:
+                if p in seen_srcs:
+                    continue
+                seen_srcs.add(p)
+                deduped_srcs.append(p)
+            srcs = deduped_srcs
+
+            # Filter out trivial source files (only includes/using/guards).
+            def _is_trivial_source(p: Path) -> bool:
+                try:
+                    text = p.read_text(encoding='utf-8', errors='ignore')
+                except Exception:
+                    return True
+                non_blank_lines = [ln.strip() for ln in text.splitlines() if ln.strip()]
+                if not non_blank_lines:
+                    return True
+                for ln in non_blank_lines:
+                    if ln.startswith('//') or ln.startswith('/*') or ln.startswith('*'):
+                        continue
+                    if ln.startswith('#include'):
+                        continue
+                    if ln.startswith('using '):
+                        continue
+                    if ln in ('{', '}', '#endif', '#if 0') or ln.startswith('#if') or ln.startswith('#define'):
+                        continue
+                    # found a non-trivial line
+                    return False
+                return True
+
+            srcs = [p for p in srcs if not _is_trivial_source(p)]
+
+            # If after filtering all source files are trivial, skip this example
+            # entirely — we don't want examples that only show includes/using.
+            if not srcs:
+                continue
```

Comment:
<img width="3440" height="1979" alt="Screenshot_20250903_003028" src="https://github.com/user-attachments/assets/ff4fe194-d5a1-40da-aac3-bd4b7b3fec67" />

Your implementation lacks some of example pages. This is why mkdocs says it can't find minimal_ui_assets.md